### PR TITLE
refactor: change package names and get rid of default SW360 dependency

### PR DESCRIPTION
**Pull request by maxhbr**
Do not use SW360 Report Generator to get rid of SW360 Dependency

### CODE REVIEW

**Comment by maxhbr** [comment]
This currently fails with: 
```
[...]
checkProxyPort[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest)  Time elapsed: 0.068 sec  <<< ERROR!
org.eclipse.sw360.antenna.api.exceptions.AntennaExecutionException: Failed to load workflow
	at org.eclipse.sw360.antenna.frontend.cli.AntennaCLISettingsReader.readSettingsToToolConfiguration(AntennaCLISettingsReader.java:201)
	at org.eclipse.sw360.antenna.frontend.cli.AntennaCLISettingsReader.readSettingsToToolConfiguration(AntennaCLISettingsReader.java:102)
	at org.eclipse.sw360.antenna.frontend.cli.AbstractAntennaCLIFrontend.init(AbstractAntennaCLIFrontend.java:66)
	at org.eclipse.sw360.antenna.frontend.cli.AbstractAntennaCLIFrontendTest.loadTestContext(AbstractAntennaCLIFrontendTest.java:48)
	at sun.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: org.eclipse.sw360.antenna.api.exceptions.AntennaConfigurationException: The config File /tmp/org.eclipse.sw360.antenna.frontend.testProjects.ExampleTestProject2933676806593394683/example-project/src/workflow.xml could not be validated againstjar:file:/home/mhuber/TNG/BSINNO/tina-review/bsinno-osm-antenna/antenna-model/target/antenna-model-1.0.0-SNAPSHOT.jar!/workflow.xsd
	at org.eclipse.sw360.antenna.xml.XMLValidator.validateXML(XMLValidator.java:60)
	at org.eclipse.sw360.antenna.workflow.WorkflowFileLoader.validateWorkflowByXSD(WorkflowFileLoader.java:134)
	at org.eclipse.sw360.antenna.workflow.WorkflowFileLoader.loadWorkflowFromClassPath(WorkflowFileLoader.java:39)
	at org.eclipse.sw360.antenna.frontend.cli.AntennaCLISettingsReader.readSettingsToToolConfiguration(AntennaCLISettingsReader.java:192)
	... 40 more
Caused by: org.xml.sax.SAXParseException; systemId: file:/tmp/org.eclipse.sw360.antenna.frontend.testProjects.ExampleTestProject2933676806593394683/example-project/src/workflow.xml; lineNumber: 54; columnNumber: 17; cvc-complex-type.2.4.a: Ungültiger Content wurde beginnend mit Element 'processors' gefunden. '{outputHandlers}' wird erwartet.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:134)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:396)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:284)
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator$XSIErrorReporter.reportError(XMLSchemaValidator.java:453)
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.reportSchemaError(XMLSchemaValidator.java:3231)
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:1791)
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:741)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:374)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2784)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:602)
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:505)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:842)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:771)
	at com.sun.org.apache.xerces.internal.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:152)
	at com.sun.org.apache.xerces.internal.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:116)
	at javax.xml.validation.Validator.validate(Validator.java:124)
	at org.eclipse.sw360.antenna.xml.XMLValidator.validateXML(XMLValidator.java:57)
	... 43 more


Results :

Tests in error: 
  testExecutionUsingMainMethod[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkAntennaContextProjectResolution[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkForUnrederedXMLproperties[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkFilesToAttach[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkConfiguredCoordinates[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  antennaFrontendReadsProject[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkParsedWorkflowForGenerators[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkConfigFiles[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkParsedWorkflowForProcessors[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkParsedConfigFiles[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  testExecution[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkParsedWorkflowForOutputHandlers[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkBooleans[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkParsedWorkflowForAnalyzers[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkProxyHost[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow
  checkProxyPort[2: Test data = example configuration](org.eclipse.sw360.antenna.frontend.AntennaBasicCLIFrontendTest): Failed to load workflow

Tests run: 64, Failures: 0, Errors: 16, Skipped: 6
```

**Comment by maxhbr** [comment]
This PR currently fails with:
```
[INFO] --- exec-maven-plugin:1.6.0:exec (gradle-build) @ antenna-gradle-plugin ---
:clean
:compileJava
:pluginDescriptors
:processResources
:classes
:jar
:javadoc
:javadocJar
:sourcesJar
:assemble
:pluginUnderTestMetadata
Note: /home/mhuber/TNG/BSINNO/tina-review/bsinno-osm-antenna/antenna-basic-assembly/antenna-gradle-plugin/src/test/java/org/eclipse/sw360/antenna/gradle/AntennaGradlePluginTest.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
:compileTestJava
:processTestResources NO-SOURCE
:testClasses
:test

org.eclipse.sw360.antenna.gradle.AntennaGradlePluginTest > testWithGradle FAILED
    java.lang.AssertionError at AntennaGradlePluginTest.java:79

2 tests completed, 1 failed, 1 skipped
:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/mhuber/TNG/BSINNO/tina-review/bsinno-osm-antenna/antenna-basic-assembly/antenna-gradle-plugin/build/reports/tests/test/index.html

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':test'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:103)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:73)
	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker$1.run(DefaultTaskGraphExecuter.java:256)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:249)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:238)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.processTask(DefaultTaskPlanExecutor.java:123)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.access$200(DefaultTaskPlanExecutor.java:79)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:104)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:98)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.execute(DefaultTaskExecutionPlan.java:663)
	at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.executeWithTask(DefaultTaskExecutionPlan.java:597)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.run(DefaultTaskPlanExecutor.java:98)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.gradle.api.GradleException: There were failing tests. See the report at: file:///home/mhuber/TNG/BSINNO/tina-review/bsinno-osm-antenna/antenna-basic-assembly/antenna-gradle-plugin/build/reports/tests/test/index.html
	at org.gradle.api.tasks.testing.AbstractTestTask.handleTestFailures(AbstractTestTask.java:611)
	at org.gradle.api.tasks.testing.AbstractTestTask.executeTests(AbstractTestTask.java:483)
	at org.gradle.api.tasks.testing.Test.executeTests(Test.java:561)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:788)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:755)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:124)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:113)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:95)
	... 33 more


* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.6/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 47s
11 actionable tasks: 11 executed
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:804)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:751)
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:313)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[...]
[INFO] Antenna Basic Maven Plugin ......................... SUCCESS [01:21 min]
[INFO] Antenna Gradle Plugin 1.0.0-SNAPSHOT ............... FAILURE [ 48.421 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 04:17 min
[INFO] Finished at: 2019-04-15T13:47:54+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (gradle-build) on project antenna-gradle-plugin: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]

```

File: antenna-workflow-steps/analyzers/antenna-csv-analyzer/pom.xml [hunk 1]
```diff
@@ -42,23 +42,27 @@
...
 			<artifactId>commons-csv</artifactId>
 		</dependency>
-		<!-- ################################ test dependencies ########################### -->
+		<!-- ################################ testing dependencies ########################### -->
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
-		</dependency>
+			<scope>test</scope>
+		</dependency	>
```
Comment (bs-ondem) [inline antenna-workflow-steps/analyzers/antenna-csv-analyzer/pom.xml]: Formatting

File: example-projects/example-project/src/workflow.xml [hunk 2]
```diff
@@ -40,31 +51,13 @@
...
-        <step>
-            <name>SW360 Report Generator</name>
-            <classHint>org.eclipse.sw360.antenna.workflow.generators.SW360DisclosureDocumentGenerator</classHint>
-            <configuration>
-                <entry key="disclosure.doc.formats" value="txt"/>
-            </configuration>
-        </step>
     </generators>
-    <processors>
-        <step>
```
Comment (bs-ondem) [inline example-projects/example-project/src/workflow.xml]: Why do you remove the Policy Engine?

**Pull request merged**